### DISCUSSION TITLE
How can I upload a PDF to an existing script

### DISCUSSION THREAD

**Discussion by tobydjones** [AdmiralGT/botc-scripts#486] [Q&A] [answered]
This script has had an update, but doesn't have a PDF:
[https://botc-scripts.azurewebsites.net/script/1945/1.0.2](https://botc-scripts.azurewebsites.net/script/1945/1.0.2)
I have generated a PDF using:
[https://script.bloodontheclocktower.com/](https://script.bloodontheclocktower.com/)
How do I add it to version 1.0.2 without creating a new version?

**Comment by AdmiralGT** [comment, marked as answer]
At the moment the UI for this isn't very clear and I plan to add an Edit button in the new year to make this a bit clearer. If you "Upload a new version", include the exact same JSON and set the version number to 1.0.2 and add the PDF it will upload the PDF for the script without making a new version.
